pkg/api/component_report.go

```diff
@@ -51,7 +51,7 @@ const (
 						ELSE 2
 					END) AS row_num
 			FROM
-				ci_analysis_us.junit
+				ci_analysis_us.qe_junit
```

Comment:
Sippy has a config file, I've been thinking if we're going to productize sippy maybe these kinds of values can go there? The dataset and project should also probably be customizable.  It could also be command-line flags but feels like sippy's already got too many.

https://github.com/openshift/sippy/blob/master/pkg/apis/config/v1/types.go

And the configs are in the config/ dir.

This would also let them setup a postgres and pull in their job data if they were interested

Reply:
I think I'd suggesting forcing the usage of a different `ci_analysis_us` and keeping the table names the same.  This PR is just looking at what it takes to rejigger enough to see data.

IOW, for this I'd redo the import

Reply:
> It could also be command-line flags but feels like sippy's already got too many.

Having read through this briefly, I think the first cut I'd make is to separate out the various modes into differnet binaries.

Reply:
I started to break things out into subcommands here, not sure if it's useful if you're intending to do something similar.  I never got around to finishing it: https://github.com/openshift/sippy/pull/807

Reply:
> I started to break things out into subcommands here, not sure if it's useful if you're intending to do something similar. I never got around to finishing it: #807

Yeah, I'd probably start somewhere like that.